source/globalCommands.py

```diff
@@ -382,8 +382,8 @@ def script_toggleSpeakCommandKeys(self,gesture):
 
 	@script(
 		# Translators: Input help mode message for toggle report font name command.
-		description=_("Toggles on and off the reporting of font changes"),
-		category=SCRCAT_DOCUMENTFORMATTING
+		description=_("Toggles on and off the reporting of font name changes"),
+		category=SCRCAT_DOCUMENTFORMATTING,

```

Comment:
Why did you change the description of this script? It is unrelated to this PR and it is not described in the PR description.
Moreover, technically, what is reported is really a font change, not the name change of a font that would remain the same but that would just be renamed. Thus this modification doesn't make sense to me.

Thus, I suggest to revert it:

```suggestion
-		description=_("Toggles on and off the reporting of font name changes"),
-		category=SCRCAT_DOCUMENTFORMATTING,
+		description=_("Toggles on and off the reporting of font changes"),
+		category=SCRCAT_DOCUMENTFORMATTING
```

Reply:
>That was discussed long time ago. Don't remember the details but I was asked to do that by the reviewer.

@jakubl7545 
I can't see a justification for this on this PR, #9507 or  #10407, so I think removing this is the best course of action. I'm going to commit this suggestion and update the changes so this can get merged.

Reply:
I see the mention here actually, but I don't think this advice should be followed. 

>Reporting of font names is present, but represented as font changes. I propose this be changed.